### DISCUSSION TITLE
Splitting up verbosity into multiple categories

### DISCUSSION THREAD

**Discussion by xmo-odoo** [pytest-dev/pytest#12770] [Ideas] [answered]
A bit of a bugbear when using pytest is that there are multiple verbosities which are unnecessarily correlated (unless I missed a way to disentangle them).

For instance the "full diff" view on failure is super convenient, but because it also triggers the verbose run log (one line per test case) it's a bit awkward when test suites are large.

This also makes it difficult to add new verbosity levels for subsystems e.g. for very large or slow test suites the dots are not that helpful but an even denser counter with some sort of timer (for the current test) could be very convenient.

**Comment by The-Compiler** [comment, marked as answer]
Duplicate of #11387

**Reply by The-Compiler** [reply]
Ah sorry, didn't realize this was a discussion rather than an issue - but yeah, you'll find more there.

**Reply by xmo-odoo** [reply]
thanks regardless for the pointer :)
